lib/Menu.php

```diff
@@ -258,6 +258,9 @@ protected function order_children( $items ) {
 					$old_menu_item = $item;
 					$item = new $this->PostClass($item);
 				}
+				if ( $item instanceof WPML_LS_Menu_Item ) {
```

Comment:
```suggestion
				if ( $item instanceof \WPML_LS_Menu_Item ) {
```

Reply:
I think this will ensure that it's looking for the constant at the root level. See this error in Scrutinizer: 

https://scrutinizer-ci.com/g/timber/timber/inspections/b3bfce39-18a1-41db-a058-3b49d203dea7/issues/files/lib/Menu.php?status=new&orderField=path&order=asc&honorSelectedPaths=0